ehrql/query_language.py

```diff
@@ -129,6 +133,12 @@ def configure_dummy_data(
         _timeout_<br>
         Maximum time in seconds to spend generating dummy data.
 
+        _oversample_<br>
+        Dummy data generation will generate a larger population and then sample from it
+        to improve the distribution of patients. This parameter controls how much larger.
+        Lower values will be faster to generate, while larger values will get closer to
+        the target distribution.
```

Comment:
I think it'd be helpful to users to give the default and an idea of what a small/large value is. i.e. 1 means no oversampling, 2 means oversampling by up to 2x the specified population size